pandas/tests/reshape/test_concat.py

```diff
@@ -2350,6 +2350,21 @@ def test_concat_datetime_timezone(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_concat_series_name_npscalar_tuple(self):
+        # GH21015
+        s1 = pd.Series({'a': 1.5}, name=np.int64(190))
+        s2 = pd.Series([], name=(43, 0))
+        result = pd.concat([s1, s2])
+        expected = pd.Series({'a': 1.5})
+        tm.assert_series_equal(result, expected)
+
+        df1 = pd.DataFrame([[1, 2], [3, 4]], columns=['a', 'b'], index=[0, 1])
```

Comment:
this is very weird to construct a DataFrame to then take a row from it. Just directly construct the series.

Reply:
will update - had added the original test case from the issue #21015